app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,108 @@
+/*
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: 2022-present Intel Corporation
+ */
+package org.omecproject.up4.cli;
+
+import com.google.common.collect.Maps;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.omecproject.up4.impl.Up4AdminService;
+import org.onlab.packet.Ip4Address;
+import org.onosproject.cli.AbstractShellCommand;
+import org.onosproject.net.behaviour.upf.UpfCounter;
+
+import java.util.Map;
+
+/**
+ * UP4 UE flow stats.
+ */
+@Service
+@Command(scope = "up4", name = "flow-stats",
+        description = "Read all UE flows stats")
+public class FlowsStatsCommand extends AbstractShellCommand {
+
+    private static final String SEPARATOR = "-".repeat(40);
+    private static final int SLEEP_TIME_MS = 200;
```

Comment:
Does this have to be a constant? Can it be a parameter? Also, 2 seconds seems a more reasonable default.